pkg/tuf/repo.go

```diff
@@ -300,11 +300,12 @@ var (
 )
 
 // GetTrustedRoot returns the trusted root for the TUF repository.
-func GetTrustedRoot() (*root.TrustedRoot, error) {
+func GetTrustedRoot(ctx context.Context) (*root.TrustedRoot, error) {
```

Comment:
The trust root resync period and cluster image policy resync flag values are [added](https://github.com/github/policy-controller/blob/release/cmd/webhook/main.go#L137-L139) to the context in `main.go`, so we can extract the value from the context.